benchmarks/src/fs2/data/benchmarks/CsvParserBenchmarks.scala

```diff
@@ -0,0 +1,42 @@
+package fs2
+package data.benchmarks
+
+import java.util.concurrent.TimeUnit
+import org.openjdk.jmh.annotations._
+
+import better.files.File
+import cats.effect.SyncIO
+
+/* Default settings for benchmarks in this class */
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@BenchmarkMode(Array(Mode.AverageTime))
+@State(org.openjdk.jmh.annotations.Scope.Benchmark)
+@Fork(value = 1)
+@Warmup(iterations = 3, time = 2)
+@Measurement(iterations = 5, time = 2)
+class CsvParserBenchmarks {
+
+  var csvContent: List[List[Char]] = _
+
+  @Setup
+  def readCsv(): Unit = csvContent = {
+    File("../../../../../benchmarks/resources/benchmark.csv").contentAsString.grouped(4096).toList.map(_.toList)
```

Comment:
Why do you read the entire file into memory? To avoid measuring I/O in the benchmark?

Reply:
Exactly. The idea behind this first, rather basic benchmark is to reveal performance regressions when changing something on the parser, not to test a real life use case. So avoiding external influence and having a baseline benchmark to get an approximation of the time really spent on parsing.

Reply:
Ok perfect. LGTM